embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -362,6 +367,16 @@ private boolean fetch(BatchSelect cursor,
                 reportRows *= 2;
             }
         } while (result.next());
+
+        if (task.getIncrementalColumn().isPresent()) {
+            for (String key : task.getIncrementalColumn().get()) {
+                if (temporaryLastRecord.containsKey(key)) {
```

Comment:
this validation logic seems invalid because incremental loading doesn't work as expected if the results don't include all of the incremental_column columns.